### PR TITLE
Fixed schema name matching for schema auth filter

### PR DESCRIPTION
**Pull request by holashchand**
Issue: https://github.com/Sunbird-RC/community/issues/777

### CODE REVIEW

File: java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SchemaAuthFilter.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 public class SchemaAuthFilter implements Filter {
     private static final Logger logger = LoggerFactory.getLogger(SchemaAuthFilter.class);
-    private static final String INVITE_URL_ENDPOINT = "/invite";
+    private static final String INVITE_URL_PATTERN = "/api/v1/([A-Za-z0-9_])+/invite(/)?";
```
Comment (srprasanna) [inline java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SchemaAuthFilter.java:15]: Why is this changed.

**Review by srprasanna — COMMENTED** [review]
@holashchand  address the review comment and add test case. Then we can close this out.

**Comment by srprasanna** [comment]
@holashchand  build is failing please resolve.

File: java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SchemaAuthFilter.java (hunk 1 above)
Comment (holashchand) [reply java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SchemaAuthFilter.java:15]: Added this to check whether the request uri is an invite endpoint. Earlier it was checking for contains "invite" which might not be well for a few cases

**Review by Sreejit-K — APPROVED** [review]

**Review by srprasanna — APPROVED** [review]
